server/src/main/java/org/kaazing/gateway/server/GatewayObserver.java

```diff
@@ -151,6 +157,18 @@ public void startingGateway(GatewayContext gatewayContext) {
             injectResources(gatewayListenerSpi, gatewayContext.getInjectables());
         }
 
+        Properties configuation = (Properties) gatewayContext.getInjectables().get("configuration");
```

Comment:
This whole section is not implemented correctly. I believe, according to [this comment](https://github.com/kaazing/roadmap/issues/1076#issuecomment-280720268) that a new `GatewayObserveFactorySpiPrototype` should be extended. Within that new class, the function `initingGateway` should be implemented and the code that you add here should be placed there. 

The `GatewayObserver` class should not be modified.